best-practices.md

```diff
@@ -2,21 +2,41 @@
 
 ## Table of Contents
 
-* [Enable CORS](#enable-cross-origin-resource-sharing-cors)
-* [Field and ID formatting](#field-and-id-formatting)
-* [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
-* [Datetime selection](#datetime-selection)
-* [Unlocated Items](#unlocated-items)
-* [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
-* [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
-* [Working with Media Types](#working-with-media-types)
-* [Static and Dynamic Catalogs](#static-and-dynamic-catalogs)
-* [Catalog Layout](#catalog-layout)
-* [Using Summaries in Collections](#using-summaries-in-collections)
-* [Use of Links](#use-of-links)
-* [Using Relation Types](#using-relation-types)
-* [Versioning for Catalogs](#versioning-for-catalogs)
-* [STAC on the Web](#stac-on-the-web)
+- **[Web Best Practices](#web-practices)**
+  - [Enable Cross-origin resource sharing (CORS)](#enable-cross-origin-resource-sharing-cors)
+  - [STAC on the Web](#stac-on-the-web)
+    - [Schema.org, JSON-LD, DCAT, microformats, etc](#schemaorg-json-ld-dcat-microformats-etc)
+    - [Deploying STAC Browser](#deploying-stac-browser)
+- **[Item Best Practices](#item-practices)**
+  - [Field and ID formatting](#field-and-id-formatting)
+  - [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
+  - [Datetime selection](#datetime-selection)
+  - [Unlocated Items](#unlocated-items)
+    - [Unrectified Satellite Data](#unrectified-satellite-data)
+    - [Data that is not spatial](#data-that-is-not-spatial)
+  - [Representing Vector Layers in STAC](#representing-vector-layers-in-stac)
+- **[Asset Best Practices](#asset-practices)**
+  - [Common Use Cases of Additional Fields for Assets](#common-use-cases-of-additional-fields-for-assets)
+  - [Working with Media Types](#working-with-media-types)
+    - [Common Media Types in STAC](#common-media-types-in-stac)
+    - [Formats with no registered media type](#formats-with-no-registered-media-type)
+- **[Catalog Best Practices](#catalog-practices)**
```

Comment:
As we may get rid of the inheritance, I'd call this "Catalog and Collection Best Practices"